docs/getting-started/installation.md

```diff
@@ -17,37 +18,162 @@ This can be done with the following command:
+        ),
+    ),
+    status="warning",
+)
+```
+
+
+## Virtual Environment (Recommended)
 
 We recommend creating a virtual environment for your project.
 
 Below are some tools you can use to manage environments:
 
 - [venv]({constants.VENV_URL})
 - [poetry]({constants.POETRY_URL})
+- [conda]({constants.CONDA_URL})
+
+```python eval
+rx.box(height=8)
+```
+
+
+
+## Installing on macOS/Linux
 
-## Installing
+Reflex's recommended package manager and environment manager for macOS and Linux are [pip]({constants.PIP_URL}) and [venv]({constants.VENV_URL}), respectively. `venv` is a part of [The Python Standard Library]({constants.PYTHON_STANDARD_LIBRARY}) and comes bundled with your installation of Python. See instructions on how to install and use `pip` below.
 
-Reflex is available as a [pip](constants.PIP_URL) package.
+
+### Install pip
+
+Install `pip`. More details about installing pip can be found in [pip's documentation]({constants.PIP_DOCS}).
+
+On a macOS:
 
 ```bash
-pip install reflex
+python -m ensurepip --upgrade
 ```
 
-## Create a Project
+```python eval
+rx.alert(
+    rx.alert_icon(),
+    rx.box(
+        rx.alert_title("Error ", rx.code("command not found: python"),),
+        rx.alert_description(
+            "On mac sometimes you need to write python3 instead of python",
+        ),
+    ),
+    status="warning",
+)
+```
 
-Installing Reflex also installs the `reflex` command line tool.
+```python eval
+rx.box(height=2)
+```
 
-Test that the install was successful by creating a new project. Replace `{app_name}` with your project name:
+On Ubuntu with Python 3:
 
 ```bash
-$ mkdir {app_name}
-$ cd {app_name}
-$ reflex init
+sudo apt-get install python3-pip
+```
+
+For other Linux distributions, see [How to install PIP for Python]({constants.HOW_TO_INSTALL_PIP}).
+
+
+
+
+### Create a new environment with Reflex
+
+1. Create a new project folder and navigate to it. Replace `{app_name}` with your project name:
+
+```bash
+mkdir {app_name}
+cd {app_name}
+```
+
+2. Create a new virtual environment in that folder and activate that environment:
+
+```bash
+python -m venv .venv
+source .venv/bin/activate
+```
+
+When you run the command above, a directory called .venv will appear in the folder {app_name} (this is a hidden folder). This directory is where your virtual environment and its dependencies are installed.
+
+
+
+3. Reflex is available as a [pip](constants.PIP_URL) package. Install Reflex in your environment:
+
+```bash
+pip install reflex
+```
+
+4. Installing Reflex also installs the `reflex` command line tool. Test that the install was successful by running:
+
+```bash
+reflex init
 ```
 
 This initializes a template app in your new directory.
 
+When you're done using this environment, type `deactivate` to return to your normal shell.
+
+
+
+
+
+```python eval
+rx.box(height=8)
+```
+
+## Installing on Windows
+
+Reflex's recommended environment manager on Windows is [Anaconda Navigator]({constants.ANACONDA_URL}).
```

Comment:
This seems a bit strong for something _we don't test internally_. I think mentioning that we also support conda, and that it might be easier for windows users would make me more comfortable.

Reply:
got rid of all of this